src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -806,9 +806,9 @@ static void check_signal_handler(int sig) {
   if (thisHandler != jvmHandler) {
     tty->print("Warning: %s handler ", os::exception_name(sig, buf, O_BUFLEN));
     tty->print_raw("expected:");
-    os::print_function_and_library_name(tty, jvmHandler, buf, O_BUFLEN);
+    os::print_function_and_library_name(tty, jvmHandler, buf, O_BUFLEN, true, true, true);
     tty->print_raw("  found:");
-    os::print_function_and_library_name(tty, thisHandler, buf, O_BUFLEN);
+    os::print_function_and_library_name(tty, thisHandler, buf, O_BUFLEN, true, true, true);
```

Comment:
These would benefit from using the same style as line 1361. No-one will remember what the three boolean flags represent.

Reply:
You are right. I reworked the style.